@xen-orchestra/backups/_runners/VmsRemote.mjs

```diff
@@ -114,6 +114,7 @@ export const VmsRemote = class RemoteVmsBackupRunner extends Abstract {
                   .run(async () => {
                     try {
                       const result = await vmBackup.run()
+                      // TODO : handle next line
                       task.success(result)
```

Comment:
@julien-f I don't really know how to handle this part with XO tasks, I have not found a similar method.

Reply:
I don't remember exactly how it works or why we did this, but I think we need to talk about this.

Reply:
We can cheat to achieve this:

```js
task.run(() => result)
```

Reply:
But I suspect it will not work because the new implementation does not allow nested runs for the same task.